designs.js

```diff
@@ -34,15 +34,11 @@ function makeList(event){
 }
 
 function copyDataToAddForm(event){
+    event.preventDefault();
```

Comment:
Same thing as line 54.
We tried to use event.preventDefault to solve the issue with form submit but you've solved it differently, so we don't need it here.